File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Declare variables
    int height = -1;

    while(height < 0 || height > 23)
    {
        // Ask for height
        printf("Height: ");

        height = get_int();
    }

    for(int i = 1; height - i >= 0; i++)
    {
        for(int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        for(int j = 0; j < i; j++)
        {
            printf("#");
        }

        printf("  ");
        for(int j = 0; j < i; j++)
        {
            printf("#");
        }

        printf("\n");
    }

}

File: pset6/mario.py
import cs50

def mario(h):
    for i in range(h):
        for j in range(h):
            if(j < h-i-1):
                print(" ",end="")
            else:
                print("#",end="")
        print("# #", end="")
        for j in range(i+1):
            print("#", end="")
        print()

def main():
    while True:
        print("Height: ", end="")
        height = cs50.get_int()
        if height >= 0:
            break
    mario(height)


if __name__ == "__main__":
    main()